Classes/Neos/Neos/Ui/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -74,7 +74,11 @@ public function renderNode(NodeInterface $node, ControllerContext $controllerCon
             $nodeInfo['uri'] = $this->uri($node, $controllerContext);
         }
 
-        foreach ($node->getChildNodes() as $childNode) {
+        $documentChildNodes = $node->getChildNodes($this->baseNodeType);
+        $contentChildNodes = $node->getChildNodes('!Neos.Neos:Document');
```

Comment:
@bwaidelich it should be all nodes needed for the content tree, no matter if rendered on the page or not. IMO `remainingChildNodes` doesn't better reflect the intent here, as these nodes are meant for the content tree, in contrast to the document tree.